static/css/base.css

```diff
@@ -238,7 +238,7 @@ input:checked + .slider:before {
     align-items: center;
     margin: 10px 0;
     font-family: 'Helvetica Neue', Arial, sans-serif;
-    font-size: 12px; /* Smaller font size */
```

Comment:
Please avoid these cosmetic changes in the PR. These changes are unrelated to this PR.

You can make these kinds of changes directly to master without review. For example, removing comments, removing/adding empty lines, removing unused imports, etc.

Reply:
Noted sir